.github/workflows/build.yaml

```diff
@@ -0,0 +1,100 @@
+name: Check PR
+
+on: pull_request
+
+jobs:
+  # Lint changed templates.
+  xlint:
+    name: Lint templates
+    runs-on: ubuntu-18.04
+
+    env:
+      PATH: '/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/usr/local/bin:/tmp/bin'
+      XLINT: '1'
+      LICENSE_LIST: common/travis/license.lst
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 200
+      - name: Fetch upstream
+        run: common/travis/fetch_upstream.sh
+      - name: Get changed templates
+        run: common/travis/changed_templates.sh
+      - name: Fetch xtools
+        run: common/travis/fetch-xtools.sh
+      - name: Run xlint
+        run: common/travis/xlint.sh
+
+  # Build changed packages.
+  build:
+    name: Build packages
+    runs-on: ubuntu-18.04
+    if: "!github.event.pull_request.draft && !contains(github.event.pull_request.title, '[ci skip]' && !contains(github.event.pull_request.body, '[ci skip]')"
+
+    container:
+      image: 'voidlinux/masterdir-${{ matrix.config.bootstrap }}:2019102301'
+      env:
+        PATH: '/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/usr/local/bin:/tmp/bin'
+        ARCH: '${{ matrix.config.arch }}'
+        BOOTSTRAP: '${{ matrix.config.bootstrap }}'
+        HOSTREPO: /hostrepo
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - { arch: x86_64, bootstrap: x86_64 }
+          - { arch: i686, bootstrap: i686 }
+          - { arch: aarch64, bootstrap: x86_64 }
+          - { arch: armv7l, bootstrap: x86_64 }
+          - { arch: x86_64-musl, bootstrap: x86_64-musl }
+          - { arch: armv6l-musl, bootstrap: x86_64-musl }
+          - { arch: aarch64-musl, bootstrap: x86_64-musl }
+
+    steps:
+      - name: Prepare container
+        run: |
+          # Sync and upgrade once (catches xbps updates)
+          xbps-install -Syu
+          # Upgrade again (in case of an xbps update)
+          xbps-install -yu
+          # Install git
+          xbps-install -y git
+
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 200
+
+      - name: Create hostrepo
+        run: ln -s "$(pwd)" /hostrepo
+
+      - name: Set mirrors
```

Comment:
I don't think we need `name: `, the script's name is descriptive enough, without name, it's defaulted to `run: `

Reply:
Agreed on most. I cleaned it up.